packages/element/src/create-interpolate-element.ts

```diff
@@ -105,13 +122,16 @@ function createFrame(
  * }
  * ```
  *
- * @param {string}                  interpolatedString The interpolation string to be parsed.
- * @param {Record<string, Element>} conversionMap      The map used to convert the string to
- *                                                     a react element.
+ * @param {string}                       interpolatedString The interpolation string to be parsed.
+ * @param {Record<string, ReactElement>} conversionMap      The map used to convert the string to
+ *                                                          a react element.
```

Comment:
Let us remove the types from JSDoc

```suggestion
 * @param  interpolatedString The interpolation string to be parsed.
 * @param  conversionMap      The map used to convert the string to
 *                            a react element.
```